keyboards/handwired/evk/info.json

```
@@ -0,0 +1,4 @@
+{
+  "manufacturer": "Yang Cui",
+  "maintainer": "Yang Cui"
```

Again, this is intended to be your GitHub username.

```suggestion
  "maintainer": "YangPiCui"
```